packages/@aws-cdk/aws-stepfunctions/lib/states/map.ts

```diff
@@ -61,6 +61,16 @@ export interface MapProps {
    */
   readonly parameters?: { [key: string]: any };
 
+  /**
+   * The JSON that you want to override the raw result of the state before ResultPath is applied.
```

Comment:
```suggestion
   * The JSON that will be used as the state's result by overriding the raw result.
   * If ResultPath is specified, it will be applied on the state's result.
```

just a suggestion, but I think we can get a little more crisp here.

Reply:
Updated to use consistent terminology as the ASL spec. Don't agree with the second sentence. ResultPath is applied to the state's raw input and injects the result into it.